src/main/java/net/sf/jabref/logic/pdf/EntryAnnotationImporter.java

```diff
@@ -0,0 +1,54 @@
+package net.sf.jabref.logic.pdf;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import net.sf.jabref.model.database.BibDatabaseContext;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.FieldName;
+import net.sf.jabref.model.entry.FileField;
+import net.sf.jabref.model.entry.ParsedFileField;
+import net.sf.jabref.model.pdf.FileAnnotation;
+
+
+/**
+ * Here all PDF files attached to a BibEntry are scanned for annotations using a PdfAnnotationImporter.
+ */
+public class EntryAnnotationImporter {
+
+    private final BibEntry entry;
+
+    /**
+     * @param entry The BibEntry whose attached files are scanned for annotations.
+     */
+    public EntryAnnotationImporter(BibEntry entry) {
+        this.entry = entry;
+    }
+
+    /**
+     * Filter files with a web address containing "www."
+     *
+     * @return a list of file parsed files
+     */
+    public List<ParsedFileField> getFilteredFileList() {
+        return FileField.parse(this.entry.getField(FieldName.FILE).get()).stream()
+                .filter(parsedFileField -> parsedFileField.getLink().toLowerCase().endsWith(".pdf"))
+                .filter(parsedFileField -> !parsedFileField.getLink().contains("www.")).collect(Collectors.toList());
```

Comment:
actually this is a one-to-one copy of what has been there before. I don't have the domain-specific insight if it makes sense.